packages/logstash/manifest.yml

```
@@ -89,16 +76,29 @@ policy_templates:
             multi: false
             required: false
             show_user: false
-      - type: cel
-        title: "Metrics (Technical Preview).\nPlease disable if using Stack Monitoring"
-        description: "Collect Metrics and stats from Logstash instances to power dedicated Logstash Dashboards in Kibana"
+      - type: logfile
+        title: "Logs"
+        description: "Collect application and slowlog logs from Logstash instances"
         vars:
-          - name: url
+          - name: condition
+            title: Condition
+            description: Condition to filter when to collect this input
             type: text
-            title: Logstash URL
-            show_user: true
+            multi: false
+            required: false
+            show_user: false
+      - type: logstash/metrics
+        title: "Metrics (Stack Monitoring)"
+        description: "Collect node metrics and stats from Logstash instances to power the Stack Monitoring application in Kibana.\n Disable if using Agent Based Monitoring."
```

```suggestion
        description: "Collect node metrics and stats from Logstash instances to power the Stack Monitoring application in Kibana.\n Disable if using Elastic Agent Based Monitoring."
```
Maybe use `Elastic Agent` here to be consistent with the `Metrics (Elastic Agent)` text from earlier?